anchore_engine/services/analyzer/analysis.py

```diff
@@ -104,6 +64,17 @@ def notify_analysis_complete(
     except Exception as err:
         logger.warn("could not marshal annotations from json - exception: " + str(err))
 
+    # Re-pull the image record before sending notifications
+    # For the case where new tags for the image were added
+    catalog_client = internal_client_for(CatalogClient, account)
+    try:
+        image_record = get_image_record(catalog_client, image_digest)
+    except Exception as err:
+        logger.warn(
+            "Cannot re-get image from catalog for image digest %s, generating notifications for already-known tags"
+            % str(image_digest)
```

Comment:
Update this to use the safe log formatting here. no need to stringify image_digest